DependencyInjection/GesdinetJWTRefreshTokenExtension.php

```diff
@@ -46,6 +46,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('gesdinet_jwt_refresh_token.single_use', $config['single_use']);
         $container->setParameter('gesdinet_jwt_refresh_token.token_parameter_name', $config['token_parameter_name']);
         $container->setParameter('gesdinet_jwt_refresh_token.doctrine_mappings', $config['doctrine_mappings']);
+        $container->setParameter('gesdinet_jwt_refresh_token.cookie', (isset($config['cookie']) ? $config['cookie'] : []));
```

Comment:
```suggestion
        $container->setParameter('gesdinet_jwt_refresh_token.cookie', $config['cookie'] ?? []);
```

PHP minimum bumped up since the original PRs, so optimize the syntax as well.